agrirouter-api-java-api/src/main/java/com/dke/data/agrirouter/api/service/onboard/OnboardingService.java

```diff
@@ -16,4 +18,10 @@
   OnboardingResponse onboard(OnboardingParameters parameters);
 
   String generateAuthorizationUrl(AuthorizationRequestParameters parameters);
+
+  /** @return The last error, if the last onboarding failed as a String */
+  String getLastErrorAsString();
+
+  /** @return The last error, if the last onboarding failed as object read from the JSON String */
+  Optional<OnboardingError> getLastError();
```

Comment:
Should be valid JavaDoc. `return` contains the description.

Reply:
Done